framework/core/Project/Math.pm

```diff
@@ -50,11 +50,23 @@ sub new {
     my $name = "commons-math";
     my $vcs  = Vcs::Git->new($PID,
                              "$REPO_DIR/$name.git",
-                             "$PROJECTS_DIR/$PID/$BUGS_CSV_ACTIVE");
+                             "$PROJECTS_DIR/$PID/$BUGS_CSV_ACTIVE",
+                             \&_post_checkout);
 
     return $class->SUPER::new($PID, $name, $vcs);
 }
 
+sub printstack {
+    my ($package, $filename, $line, $subroutine, $hasargs, $wantarray, $evaltext, $is_require, $hints, $bitmask, $hinthash);
+    my $i = 1;
+    my @r;
+    while (@r = caller($i)) {
+        ($package, $filename, $line, $subroutine, $hasargs, $wantarray, $evaltext, $is_require, $hints, $bitmask, $hinthash) = @r;
+        print "$filename:$line $subroutine\n";
+        $i++;
+    }
+}
+
```

Comment:
Can we remove this debugging function?

Reply:
If useful for debugging in general, perhaps move this to the Utils module?

Reply:
I will move to Utils and make change the function name to print_perl_call_stack.